newIDE/app/src/ObjectsList/index.js

```diff
@@ -143,6 +147,8 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
       resourceManagementProps,
       onSelectAllInstancesOfObjectInLayout,
       onDeleteObject,
+      renamedObjectWithContext,
+      setRenamedObjectWithContext,
```

Comment:
I'm a bit confused by passing down props that are name `setXXX` because you don't really know when/why you should use them and what it does.
Should this rather be called something more specific?
Like `onRenameStart()` or `onRenameFinish()`.
It makes more sense from the component's point of view what it should do with it.

Reply:
Agree that it should be called onXxx. or onSetXxx.
THis is because the children is not aware of what it does exactly, but the name gives an indication about "when" it must be called ("on" something happening...).
The parent is the one responsible for doing something.